src/Controller/ExampleController.php

```diff
@@ -24,59 +24,57 @@ public function indexAction()
     {
         return $this->render('PsPdfBundle:Example:index.html.twig');
     }
-    
+
     public function usingFacadeDirectlyAction()
     {
         $facade = $this->get('ps_pdf.facade');
         $response = new Response();
-        $this->render('PsPdfBundle:Example:usingFacadeDirectly.pdf.twig', array(), $response);
-        
+        $this->render('PsPdfBundle:Example:usingFacadeDirectly.pdf.twig', [], $response);
+
         $xml = $response->getContent();
-        
+
         $content = $facade->render($xml);
-        
-        return new Response($content, 200, array('content-type' => 'application/pdf'));
+
+        return new Response($content, 200, ['content-type' => 'application/pdf']);
     }
 
     /**
-     * Possible custom headers and external stylesheet file
-     * 
+     * Possible custom headers and external stylesheet file.
+     *
      * @Pdf(
-     * 	headers={"Expires"="Sat, 1 Jan 2000 12:00:00 GMT"}, 
+     * 	headers={"Expires"="Sat, 1 Jan 2000 12:00:00 GMT"},
      * 	stylesheet="PsPdfBundle:Example:pdfStylesheet.xml.twig",
      *  enableCache=true
      * )
      */
     public function usingAutomaticFormatGuessingAction($name)
     {
         $format = $this->get('request')->get('_format');
-        
-        return $this->render(sprintf('PsPdfBundle:Example:usingAutomaticFormatGuessing.%s.twig', $format), array(
+
+        return $this->render(sprintf('PsPdfBundle:Example:usingAutomaticFormatGuessing.%s.twig', $format), [
             'name' => $name,
-        ));
+        ]);
     }
-    
+
     /**
-     * Standard examples of PHPPdf library
+     * Standard examples of PHPPdf library.
      */
     public function examplesAction()
     {
         $kernelRootDir = $this->container->getParameter('kernel.root_dir');
-        
-        $propablyPhpPdfExamplesFilePaths = array($kernelRootDir.'/../vendor/PHPPdf/examples/index.php', $kernelRootDir.'/../vendor/psliwa/php-pdf/examples/index.php');
 
-        foreach($propablyPhpPdfExamplesFilePaths as $propablyPhpPdfExamplesFilePath)
-        {
-            if(file_exists($propablyPhpPdfExamplesFilePath))
-            {
+        $propablyPhpPdfExamplesFilePaths = [$kernelRootDir.'/../vendor/PHPPdf/examples/index.php', $kernelRootDir.'/../vendor/psliwa/php-pdf/examples/index.php'];
```

Comment:
You can use `kernel.project_dir` instead of `kernel.root_dir`
```suggestion
        $kernelProjectDir = $this->container->getParameter('kernel.project_dir');
        $propablyPhpPdfExamplesFilePaths = [$kernelProjectDir.'/vendor/PHPPdf/examples/index.php', $kernelProjectDir.'/vendor/psliwa/php-pdf/examples/index.php'];
```